Content/Items/Hell/Accessories.FuryInABottle.cs

```diff
@@ -0,0 +1,98 @@
+﻿using StarlightRiver.Content.Items.BaseTypes;
+using StarlightRiver.Content.NPCs.Actors;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.Items.Hell
+{
+	internal class FuryInABottle : SmartAccessory
+	{
+		public int charge;
+
+		public override string Texture => AssetDirectory.HellItem + Name;
+
+		public FuryInABottle() : base("Fury in a Bottle", "Allows a short double jump\nRecharges for every 100 damage dealt") { }
+
+		public override void Load()
+		{
+			StarlightPlayer.OnHitNPCEvent += GainCharge;
+			StarlightPlayer.OnHitNPCWithProjEvent += GainChargeProj;
+		}
+
+		public override void SafeSetDefaults()
+		{
+			Item.rare = ItemRarityID.Orange;
+			Item.value = Item.sellPrice(0, 1, 0, 0);
+		}
+
+		private void GainCharge(Player player, Item Item, NPC target, NPC.HitInfo hit, int damageDone)
+		{
+			if (Equipped(player) && !player.GetJumpState<FuryJump>().Available)
+			{
+				var instance = GetEquippedInstance(player) as FuryInABottle;
+				instance.charge += damageDone;
+			}
+		}
+
+		private void GainChargeProj(Player player, Projectile proj, NPC target, NPC.HitInfo hit, int damageDone)
+		{
+			if (Equipped(player) && !player.GetJumpState<FuryJump>().Available)
```

Comment:
this probably desyncs in multiplayer and it probably only needs to send the onhit packets like other ones. otherwise you could send a packet in the UpdateAccessory when it charges up that tells other clients that its charged and ready.

i.e. add 
Player.GetModPlayer<StarlightPlayer>().SetHitPacketStatus(shouldRunProjMethods: false);;

like BarbedKnife onhits as an example